src/components/ListPanel/users/SelectReviewerListItem.vue

```diff
@@ -33,18 +33,19 @@
 					</span>
 				</div>
 
-				<div class="listPanel__itemSubtitle">
+				<div
+					v-if="item.affiliation || item.orcid"
+					class="listPanel__itemSubtitle"
+				>
 					<div
-						v-if="item.affiliation || item.orcid"
+						v-if="item.affiliation"
 						class="listPanel__item--reviewer__affiliation"
```

Comment:
Hi @taslangraham , I tested this on storybook and your fix works. 
Just thought that we should address the long affiliation names. Currently, the rest of the text disappears:
<img width="896" height="357" alt="image" src="https://github.com/user-attachments/assets/893df454-0490-4df9-b0ce-104b14a74122" />

We can fix this by adding these classes:
```suggestion
						class="listPanel__item--reviewer__affiliation whitespace-normal break-words"
```

Then the affiliation name will render like this:
<img width="890" height="262" alt="image" src="https://github.com/user-attachments/assets/5f4d7124-2696-4d5f-8c75-230721445c12" />